docs/jupiterone.md

```diff
@@ -96,21 +96,30 @@ https://github.com/JupiterOne/sdk/blob/main/docs/integrations/development.md
 
 The following entities are created:
 
-| Resources | Entity `_type` | Entity `_class` |
-| --------- | -------------- | --------------- |
-| Account   | `acme_account` | `Account`       |
-| User      | `acme_user`    | `User`          |
-| UserGroup | `acme_group`   | `UserGroup`     |
+| Resources          | Entity `_type`             | Entity `_class` |
+| ------------------ | -------------------------- | --------------- |
+| Account            | `asana_account`            | `Account`       |
+| Project            | `asana_project`            | `Project`       |
+| Project Membership | `asana_project_membership` | `AccessRole`    |
+| Team               | `asana_team`               | `Team`          |
+| User               | `asana_user`               | `User`          |
+| Workspace          | `asana_workspace`          | `Organization`  |
```

Comment:
@llravelo Do you know if there can be more than one workspace for the single account/auth? If no, `asana_workspace` feels like perfect root node (even right now if you take a look at graph it feels more centralized than `asana_account`) but it really depends on this.

![image](https://user-images.githubusercontent.com/8009142/164530543-c652b875-c59e-4eff-a36b-37f633a93ecd.png)

Reply:
@eXtremeX Yes, it is possible to have more than one workspace. For example, I just created a new workspace with sample members. This is how it looks like now:

![image](https://user-images.githubusercontent.com/22865398/164581323-97e70dda-42f7-4dc2-aa03-0085ff1ee648.png)